data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
 			`	"When your name came up in the study," he says, "I asked around and discovered that you have helped us in other ways as well. The other prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall henceforth be given permission to purchase our ships and technology."`
 			choice
 				`	"I'm honored. Thank you."`
-					decline
+					goto onwards
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
-				decline
+			`	"I do not have anything at the moment," he says, "but I am sure other Remnant will have tasks you can help with, now that you have demonstrated your trustworthiness. We have been keeping track of you, and we are impressed with your curiosity, perseverance, and ability to survive where most would be destroyed. All three are valued characteristics among the Remnant, and thus many will welcome your assistance on their projects.`
+			label onwards
+			`	As he talks, he gestures for you to walk with him towards a nearby hall. "While you are welcome to become one of us, we suspect that you are a citizen of the galaxy - wandering hither and yon building coalitions and enmities with species and groups across the breadth of accessible space, unfettered by conventional allegiances. As such, we hope that you will continue to explore and stop by to let us know what you have found. We ask only that you keep the secret with which you are trusted, and bring us word of anything that might influence humanity's long-term survival."`
+			`	You follow the prefect into the hall, and discover that it is one of the many cafeterias that seem to be common in Remnant starports. He picks up two trays from the window and leads you to an empty table.`
+			choice
+				`	"What secret do you want me to keep?`
+					goto secret
+				`	"What constitutes 'things that influence humanity's survival?"`
+			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity; or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines, or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships, or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite.`
```

Comment:
In case the player has already finished the missions where Automata invade the Far North, this would be a good place for the player to tell the guy about it. :)

But maybe you were intending to do that in another PR?

Reply:
The plan is that - eventually - there will be a basically four categories of missions:
1) Core Remnant missions - these are the ones that build up to and lay the foundations for the main campaign. Everything that feeds into licenses and lays important ground work for the start of the campaign.
2) Campaign missions - The Cognizance missions are the start of the Remnant campaign. These will be the main, mostly single path, long story set of missions that are mostly independent of what goes on in the rest of the galaxy.
3) Side strings - Following the pattern of things like the Expanded Horizons, Broken Jump Drives, and Crystal Research, I'm planning on plenty of small strings off the side of the main campaign in which the Remnant explore various things and learn more about the galaxy.
4) Remnant Reactions - This set of missions that are currently being planned are kind of like the side strings of 3, except they are tied directly to events and things in the larger galaxy. The Automata invasion of the far North and the Exiles stealing Automata tech are two major ones that I've got some missions planned for. Likewise for the Pug invasion, discovery of the Wanderers, and a variety of other major plot events. Eventually, I'd like to give the player the option of regularly checking in with the Remnant to tell them about various activities and stuff throughout the galaxy.